Steps/Steps.js

```diff
@@ -109,10 +109,10 @@ const StepsBase = kind({
 
 		/**
 		 * @type {String}
-		 * @default 'wizardPanels'
+		 * @default 'highlightCompletedStep'
 		 * @private
 		 */
-		layout: PropTypes.string,
+		layout: PropTypes.oneOf(['highlightCompletedStep', 'highlightCurrentStep']),
```

Comment:
Please add proper documentation for this prop.
And as per my understanding, this prop might be clearer if it's a boolean. Let's rename this to `hightlightCurrentOnly`.

Reply:
It's fixed.